spec/checkout_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'checkout'
+
+describe Checkout do
+  describe "#send_notification" do
+    it "should send an SMS with the delivery time window" do
+      checkout = Checkout.new
+      # expect(checkout.send_notification).to do something with Twilio

```

Comment:
By digging a bit into the Twilio library, you could have something like...
```ruby
expect_any_instance_of(Twilio::REST::Api::V2010::AccountContext::MessageList).to receive(:send).with({ from: expected_from, to: expected_to, body: expected_body })
```

to assert that the library is called with the right parameters when `checkout.send_notification` is called